tests/hooks.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * A Root Hook Plugin is loaded via --require which “registers” one or more root hooks to be used across all test files.
+ */
+require('dotenv').config({ path: process.cwd() + '/tests/.env' });
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const MadWalletJS = require("../index.js");
+const madWallet = new MadWalletJS(42, process.env.RPC || "http://127.0.0.1:8884/v1/");
+
+exports.mochaHooks = {
+	async beforeAll() {
+		try {
+			// Check if PRIVATE_KEY is empty in .env file
+			if(!process.env.PRIVATE_KEY) {
+				throw 'You need to setup a funded Private Key in your .env file to run tests properly.';
+			}
+
+			// add 2 bn curve addresses
```

Comment:
Code  from `bls_flow.js` moved here.

Reply:
In reality this block is not doing anything. All these functions have tests in their own suite.
If it's not a necessity to log out examples I think it can be removed. What do you think?